.github/dependabot.yml

```diff
@@ -11,3 +11,8 @@ updates:
     - dependency-name: "github.com/cosmos/ibc-go"
       update-types: ["version-update:semver-major", "version-update:semver-minor"]
   open-pull-requests-limit: 10
+- package-ecosystem: cargo
+  directory: "/"
```

Comment:
Should this instead be pointing to https://github.com/osmosis-labs/osmosis/tree/main/cosmwasm

since we don't seem to have any cargo files in the root of the repo?

Reply:
maybe?  I think it will find all of them in the repo, but might as well?  

Not 100% sure actually